fe/be-java-extensions/java-common/src/main/java/org/apache/doris/common/jni/vec/ColumnType.java

```diff
@@ -258,6 +258,9 @@ public static ColumnType parseType(String columnName, String hiveType) {
             case "largeint":
                 type = Type.LARGEINT;
                 break;
+            case "long":
```

Comment:
we should generate hive compatible types.

Reply:
done